cypress/integration/visitorCanViewLocalNews.feature.js

```diff
@@ -0,0 +1,39 @@
+const stubLocation = require("../support/stubLocation");
+
+
+describe("visitors can veiw articles ", () => {
+  context('based on their based on sweden location', () => {
+    before(() => {
+      cy.server()
+      cy.route({
+        method: "GET",
+        url: "http://localhost:3000/api/v1/articles?latitude=60&longitude=18",
```

Comment:
just a question, why we use ? here at the end of url?

Reply:
because of the query parameters that follow